### PR TITLE
:lipstick: added option to keep style on drilled paths

### PR DESCRIPTION
**Pull request by Brynjulf**
dependent on https://github.com/equinor/petec-well-experience/pull/1284 to be able to test this properly

### CODE REVIEW

**Review by arkadiy93 — CHANGES_REQUESTED** [review]
you need to bump the version of the package

File: src/organisms/SideBar/MenuItem.tsx [hunk 1]
```diff
@@ -3,13 +3,20 @@ import { forwardRef, HTMLAttributes, MouseEvent, useMemo } from 'react';
...
 import { useSideBar } from 'src/providers/SideBarProvider';
 
 export type MenuItemProps = {
   currentUrl?: string;
   disabled?: boolean;
+  /**
+   * This is useful for routes that drills into a path,
+   * by still highlighting the original path and not disable "back to start" by click on the same link again
+   * eg. navigating to /collections and clicking on an item to go into /collections/:id and so on
+   */
```
[the remark this answers is not in the document]
Comment (Brynjulf) [reply src/organisms/SideBar/MenuItem.tsx:18]: This is the js doc format, when you look at the property while using the component this comment should accurately show when you are looking at what the property should do

[the remark this answers is not in the document]
Comment (arkadiy93) [reply src/organisms/SideBar/MenuItem.tsx:18]: Seems like it does 😸 

image

[the remark this answers is not in the document]
Comment (Brynjulf) [reply src/organisms/SideBar/MenuItem.tsx:18]: Screenshot 2024-09-03 at 14 31 01

[the remark this answers is not in the document]
Comment (aslakihle) [reply src/organisms/SideBar/MenuItem.tsx:18]: ok, nice! 
Was thinking of this, but guess its a way to do the same thing:
image

**Review by aslakihle — APPROVED** [review]

**Review by arkadiy93 — APPROVED** [review]

**Pull request merged**